modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Inline toolbars flicker when switching between editors #TINY-8594
+- Multiple inline toolbars are shown after scrolling down and up #TINY-8503
```

Comment:
```suggestion
- Multiple inline toolbars were shown if focused too quickly #TINY-8503
```
This wasn't caused by scrolling up and down. That was the spellchecker part which was causing the editor to be focused. Also just as a heads up, changelog entries should be past tense.